doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -139,18 +140,22 @@ Pelion
 MCUboot
 =======
 
-The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``2fce9769b1``, plus some |NCS| specific additions.
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``7a51968``, plus some |NCS| specific additions.
 
 The code for integrating MCUboot into |NCS| is located in :file:`ncs/nrf/modules/mcuboot`.
 
 The following list summarizes the most important changes inherited from upstream MCUboot:
 
-* No changes yet
+* added AES support for image encryption (mbed TLS based).
+* boot serial: Ported encoding to use cddl-gen (which removes dependency on TinyCBOR).
```

Comment:
```suggestion
* MCUboot serial: Ported encoding to use the cddl-gen module (which removes dependency on the `TinyCBOR`_ library).
```
Isn't this MCUboot serial? Also, is this related to the CONFIG_CDDL_GEN Kconfig?